relayer/src/ethereum_checkpoints/tests/mod.rs

```diff
@@ -497,10 +504,13 @@ async fn sync_update_requires_replaying_back() -> Result<()> {
 
     let (_message_id, payload, _value) = listener.reply_bytes_on(message_id).await?;
     let result_decoded = HandleResult::decode(&mut &payload.unwrap()[..]).unwrap();
-    assert!(matches!(
-        result_decoded,
-        HandleResult::SyncUpdate(Err(sync_update::Error::ReplayBackRequired { .. }))
-    ));
+    assert!(
+        matches!(
+            result_decoded,
+            HandleResult::SyncUpdate(Err(sync_update::Error::ReplayBackRequired { .. }))
+        ),
+        "result_decoded = {result_decoded:?}"
```

Comment:
`Debug` string will be printed in case of failure even without formatting, isn't it?